exercises/levenshtein-distance/canonical-data.json

```
@@ -0,0 +1,65 @@
+{
+  "exercise": "levenshtein-distance",
+  "version": "1.0.0",
+  "cases": [
+    {
+      "description": "Calculate Levenshtein distance",
+      "cases": [
+        {
+          "description": "mine_to_times",
+          "property": "distance",
+          "input": {
+            "phrase": "mine",
+			"phrase": "times"
+          },
+          "expected": "3"
+        },
+		{
+          "description": "_to_abcd",
+          "property": "distance",
+          "input": {
+            "phrase": "",
+			"phrase": "abcd"
```

```suggestion
            "phrase": "abcd"
```
Replace tabs with spaces.